samples/mem_tg/tg_test.h

```diff
@@ -66,103 +70,109 @@ class tg_test : public test_command
         return (double)(xfer_bytes) / ((1000.0 / (double)tg_exe_->mem_speed_ * (double)num_ticks));
     }
 
-    void tg_perf () {
-      uint32_t mem_ch_offset = (tg_exe_->mem_ch_) << 0x3;
+    void tg_perf (mem_tg *tg_exe_) {
+      uint32_t mem_ch_offset = (std::stoi(tg_exe_->mem_ch_[0])) << 0x3;
       uint64_t num_ticks = tg_exe_->read64(MEM_TG_CLOCKS + mem_ch_offset);
       std::cout << "Mem Clock Cycles: " << std::dec << num_ticks << std::endl;
```

Comment:
Regarding the use of std::cout, I see that there are logger calls as well.  Can we standardize on a single method?

Reply:
I'm not sure if it was on purpose or not but it seemed like mem_tg used logger and tg_test used cout when I picked the code up. The effect is that things written from mem_tg have the logger preamble and the things written in tg_test don't. I'm happy to standardize it if it seems like this delineation doesn't do much.

![image](https://github.com/OFS/opae-sdk/assets/71091164/4b924386-fed2-4bd1-b2e8-3f17f3b3bc1f)